src/utils/address.ts

```diff
@@ -15,10 +15,10 @@ export const compareAddress = (addressOne: Address | undefined, addressTwo: Addr
   if (addressOne?.addressLine2?.toLowerCase().trim() !== addressTwo?.addressLine2?.toLowerCase().trim()){
     return false;
   }
-  if (addressOne?.country.toLowerCase().trim() !== addressTwo?.country?.toLowerCase().trim()){
+  if (addressOne?.country?.toLowerCase().trim() !== addressTwo?.country?.toLowerCase().trim()){
```

Comment:
(addressOne?.country?.toLowerCase().trim() ?? '') !== (addressTwo?.country?.toLowerCase().trim() ?? '')

Reply:
The addresses coming from the UI would have to have these fields so flagging a change.